WebDriverAgentLib/Utilities/FBXPath-Private.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "FBXPath.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FBXPath ()
+
+/**
+ Gets xmllib2-compatible XML representation of n XCElementSnapshot instance
+ 
+ @param root the root element to execute XPath query for
+ @param writer the correspondig libxml2 writer object
+ @param withElementStore an empty dictionary to store indexes mapping
+ @return zero if the method has completed successfully
+ */
++(int)getSnapshotAsXML:(XCElementSnapshot *)root withWriter:(xmlTextWriterPtr)writer withElementStore:(NSMutableDictionary *)elementStore;
```

Comment:
According to Apple convention "With"/"For" should be used only in first parameter any following should have plain name eg.
`+ (int)matchXMLForSnapshot: write: elementStore:`

Please fix names everywhere.